napari/components/_viewer/model.py

```diff
@@ -360,13 +366,13 @@ def _calc_layers_ranges(self):
         """
 
         ndims = self._calc_layers_num_dims()
-        ranges = [(inf, -inf, inf)]*ndims
+        ranges = [(inf, -inf, inf)] * ndims
 
         for layer in self.layers:
             layer_range = layer.range[::-1]
-            ranges = [(min(a, b), max(c, d), min(e, f)) for
-                      (a, c, e), (b, d, f) in zip_longest(ranges, layer_range,
-                      fillvalue=(inf, -inf, inf))]
+            ranges = [(min(a, b), max(c, d), min(e, f))
+                      for (a, c, e), (b, d, f) in zip_longest(
+                          ranges, layer_range, fillvalue=(inf, -inf, inf))]
 
```

Comment:
for this section `yapf` seems more pythonic but new line in zip_longest argument is cringe